documents/charter.md

```diff
@@ -63,7 +63,7 @@ The proposed deliverables for SCONEPRO are as follows:
     * connection establishment, communication signaling and data format (container to share network information)
     * specify and justify privacy and security requirements
 
-* Develop a standard track specification based on SCONEPRO protocol to communicate network properties applicable to streaming video from the SCONEPRO monitor to the client(s). Its contents might include
+* Develop a standard track specification based on SCONEPRO protocol to specify a format for network properties applicable to streaming video to be communicated from the SCONEPRO monitor to the client(s). Its contents might include
```

Comment:
@mirjak, thanks for submitting this. IMO, the problem is that we merged two deliverables (one for a protocol, the other for the format), and I was paying attention to the sub-bullet level of detail, but didn't notice that the bullet-level of detail was incomplete. 

I think the correct change would be this.

```suggestion
* Develop a standard track specification describing a SCONEPRO protocol to communicate network properties applicable to streaming video from the SCONEPRO monitor to the client(s), and specifying a format for those applicable network properties.  Its contents might include
```